tests/golden_tests/run_single_files/readback_hvm1_main.hvm

```diff
@@ -0,0 +1,3 @@
+// Main = λa Main
+// FIXME: why does this exist?
```

Comment:
hvm-lang's default entrypoint is `main`, but for backwards compatibility we also accept `Main`, the entrypoint of the old hvm.
We don't allow programs to have both `main` and `Main` functions because it would be ambiguous.

Since hvm-core only looks for `@main`, we compile both `main` and `Main` to `@main`.

This is why this test exists

Reply:
Great, thanks for the explanation! I commented out that file when the way I made hvm-lang handle entrypoints prevented this test from working. With the changes to `entrypoint`, the original version of the test works out of the box, so this is no longer necessary. Commit e0a0422 uncomments the test and moves it to the `run_file` directory.